source/_data/themes.yml

```diff
@@ -2853,3 +2853,14 @@
     - seo
     - 设计
     - literary
+- name: Mdm
+  description: 一款MaterialDesign风格主题，简洁，轻巧，麻麻再也不用担心打开博客速度慢了
+  link: https://github.com/TonyChenn/mdm
+  preview: https://tonychenn.cn
+  tags:
+    - Material Design
+    - Light
+    - Faster
```

Comment:
The "Light" here means bright background or "lightweight"? If it's lightweight, "fast" tag is not needed.